dipy/direction/pmf.pyx

```diff
@@ -14,15 +15,21 @@ from dipy.core.interpolation cimport trilinear_interpolate4d_c
 cdef class PmfGen:
 
     def __init__(self,
-                 double[:, :, :, :] data):
-        self.data = np.asarray(data,  dtype=float)
+                 double[:, :, :, :] data,
+                 object sphere=default_sphere):
```

Comment:
I'm not comfortable with this. I would prefer having no default value for the sphere. The only sphere that makes sense to use here is the precise sphere corresponding to the spherical data point of the input `data`. Any other sphere will results in an erroneous interpretation of the pmfs. 

If having a default sphere value is necessary, I would advocate for using `None`, and throwing warnings in functions requiring the sphere (i.e. `get_pmf_value(.)`).